### DISCUSSION TITLE
How to convert the REDS dataset into LMDB format for EDVR training?

### DISCUSSION THREAD

**Discussion by amy1626** [open-mmlab/mmediting#1280] [Datasets]
How to convert the REDS dataset into LMDB format for EDVR training?

**Comment by wangruohui** [comment]
Please see if this doc helps 
https://mmediting.readthedocs.io/en/latest/_tmp/sr_datasets.html#prepare-lmdb-dataset-for-reds

**Comment by amy1626** [comment]
> Please see if this doc helps https://mmediting.readthedocs.io/en/latest/_tmp/sr_datasets.html#prepare-lmdb-dataset-for-reds

Sorry, maybe I didn't express myself clearly.
What I want to know more is how to modify the relevant configuration files to train the edvr model after converting to lmdb format.
I'm confused about this.

**Comment by wangruohui** [comment]
How about this https://github.com/open-mmlab/mmediting/blob/master/mmedit/datasets/sr_lmdb_dataset.py#L9 ?
You can configure the type of dataset to be `SRLmdbDataset` and fill arguments following the docstring.

**Comment by amy1626** [comment]
After configuring the type of the dataset to be SRLmdbDataset, a series of problems appeared.
such as:

TypeError: SRLmdbDataset: __init__() got an unexpected keyword argument 'val_partition'/'num_input_frames'

and 

![image](https://user-images.githubusercontent.com/105699886/169200166-04c556a2-c0a8-49eb-bf47-0debaa2153bf.png)

**Comment by zengyh1900** [comment]
Please check this issue. @Z-Fran

**Comment by Z-Fran** [comment]
Sorry for the late reply. SRLmdbDataset may not support video datasets well. You can try using [MMEditing 2.0](https://github.com/open-mmlab/mmediting/tree/1.x).
